vital/types/point.h

```diff
@@ -38,36 +39,48 @@
 #ifndef KWIVER_VITAL_POINT_H_
 #define KWIVER_VITAL_POINT_H_
 
-#include <vital/vital_config.h>
-#include <vital/vital_export.h>
 #include <vital/types/covariance.h>
 #include <vital/types/vector.h>
 
+#include <vital/vital_config.h>
+#include <vital/vital_export.h>
+
 #include <memory>
 
 namespace kwiver {
+
 namespace vital {
 
 template < unsigned N, typename T >
 class VITAL_EXPORT point
 {
 public:
   using vector_type = Eigen::Matrix< T, N, 1 >;
-  using covariance_type = covariance_<N, float>;
+  using covariance_type = covariance_< N, float >;
+
+  point() {}
+  point( vector_type const& v, covariance_type const& c = {} )
+    : m_value{ v }, m_covariance{ c } {}
+
+  template < unsigned K = N,
```

Comment:
I don't fully understand this syntax, but I trust you that this is correct.  I assume the compiler can't resolve the template if you don't introduce this additional `K` template parameter, right?

Reply:
Right; `enable_if` is wonky. (See also https://eigen.tuxfamily.org/bz/show_bug.cgi?id=1768; you aren't the only one looking cross-eyed at correct usage of `enable_if` :slightly_smiling_face:.)